subsys/logging/Kconfig.backends

```diff
@@ -56,6 +56,12 @@ config LOG_BACKEND_UART_OUTPUT_DICTIONARY_BIN
 
 endchoice
 
+config LOG_BACKEND_UART_OUTPUT_DEFAULT
+	int
+	default 0 if LOG_BACKEND_UART_OUTPUT_TEXT
+	default 1 if LOG_BACKEND_UART_SYST_ENABLE
```

Comment:
The name LOG_BACKEND_UART_SYST_ENABLE is inconsistent with the others. I suggest changing this to follow the LOG_BACKEND_UART_OUTPUT_xxx style.

Reply:
Changed the LOG_BACKEND_UART_SYST_ENABLE to LOG_BACKEND_UART_OUTPUT_SYST. Will do the same for other backends when will add support for them after initial reviews on the design. Thanks